content/en/building/contact-management/contacts.md

```diff
@@ -81,9 +78,9 @@ From profiles, users can edit contact information, take actions, and, if viewing
 
 The top card on all profiles contains general information for the contact. All the fields shown in this summary card are configurable.
```

Comment:
these larger contact summary screenshots are an improvement over the old ones - nice!